examples/main.go

```diff
@@ -217,6 +219,7 @@ func main() {
 	m["mastodon"] = "Mastodon"
 	m["wecom"] = "WeCom"
 	m["zoom"] = "Zoom"
+	m["patreon"] = "Patreon"
```

Comment:
Sorry I'm just making a bunch of one-off comments, but could you add the two lines you added into this file into their respective lists in alpha order?